docs/reference/quickstart/index.asciidoc

```diff
@@ -9,23 +9,34 @@ Unless otherwise noted, these examples will use queries written in <<query-dsl,Q
 == Requirements
 
 You'll need a running {es} cluster, together with {kib} to use the Dev Tools API Console.
-Get started <<run-elasticsearch-locally,locally in Docker>> , or see our <<elasticsearch-intro-deploy,other deployment options>>.
+Run the following command in your terminal to set up a <<run-elasticsearch-locally,single-node local cluster in Docker>>:
+
+[source,sh]
+----
+curl -fsSL https://elastic.co/start-local | sh
+----
+// NOTCONSOLE
+
+Alternatively, see our <<elasticsearch-intro-deploy,other deployment options>>.
```

Comment:
The [accessibility guidelines](https://docs.elastic.dev/content-design/accessibility#terms) recommend avoiding the word "see".

```suggestion
Alternatively, refer to our <<elasticsearch-intro-deploy,other deployment options>>.
```